src/__tests__/lib/command/output-builder.test.ts

```diff
@@ -1,41 +1,59 @@
-import yargs, { Argv } from 'yargs'
-
-import { CalculateOutputFormatFlags, calculateOutputFormat, calculateOutputFormatBuilder, jsonFormatter, yamlFormatter } from '../../../lib/command/output.js'
-import { BuildOutputFormatterFlags, buildOutputFormatter, buildOutputFormatterBuilder } from '../../../lib/command/output-builder.js'
+import { jest } from '@jest/globals'
+
+import { Argv } from 'yargs'
+
+import {
+	CalculateOutputFormatFlags,
+	OutputFormatter,
+	calculateOutputFormat,
+	calculateOutputFormatBuilder,
+	jsonFormatter,
+	yamlFormatter,
+} from '../../../lib/command/output.js'
+import { BuildOutputFormatterFlags } from '../../../lib/command/output-builder.js'
 import { SimpleType } from '../../test-lib/simple-type.js'
 import { CLIConfig } from '../../../lib/cli-config.js'
 import { SmartThingsCommandFlags } from '../../../lib/command/smartthings-command.js'
 
 
-jest.mock('../../../lib/command/output.js')
+const calculateOutputFormatMock: jest.Mock<typeof calculateOutputFormat> = jest.fn()
+const calculateOutputFormatBuilderMock: jest.Mock<typeof calculateOutputFormatBuilder> = jest.fn()
+const jsonFormatterMock: jest.Mock<typeof jsonFormatter<SimpleType>> = jest.fn()
+const yamlFormatterMock: jest.Mock<typeof yamlFormatter<SimpleType>> = jest.fn()
+jest.unstable_mockModule('../../../lib/command/output.js', () => ({
+	calculateOutputFormat: calculateOutputFormatMock,
+	calculateOutputFormatBuilder: calculateOutputFormatBuilderMock,
+	jsonFormatter: jsonFormatterMock,
+	yamlFormatter: yamlFormatterMock,
+}))
+
+
+const { buildOutputFormatter, buildOutputFormatterBuilder } = await import('../../../lib/command/output-builder.js')
+
 
 test('buildOutputFormatterBuilder', () => {
```

Comment:
This is the only unit test I've written so far for a "builder" function. I'm not sure if it's really worth it but I might try a couple more and see how they look. If nothing else, it's nice to get that sweet sweet 100% coverage. 😄